### DISCUSSION TITLE
OpenClash支持代理7844端口吗？

### DISCUSSION THREAD

**Discussion by FishHawk** [vernesong/OpenClash#3271] [Q&A] [answered]
我尝试用OpenClash代理Cloudflare tunnel的流量，但是日志里没有7844端口的记录，请问OpenClash支持代理7844端口吗？

**Comment by Plutonium141** [comment, marked as answer]
在插件设置→流量控制下，有一个仅允许常用端口流量选自定义就可以代理你想要的端口了

**Reply by FishHawk** [reply]
谢谢，我去试试。
